test/parallel/test-async-hooks-http-parser-destroy.js

```diff
@@ -16,7 +16,7 @@ const createdIds = [];
 const destroyedIds = [];
 async_hooks.createHook({
   init: common.mustCallAtLeast((asyncId, type) => {
-    if (type === 'HTTPPARSER') {
+    if (type === 'HTTPINCOMINGMESSAGE' || type === 'HTTPCLIENTREQUEST') {
```

Comment:
Does `HTTPCLIENTREQUEST` ever actually happen here? My messing around with this for another PR has that never showing up.

Reply:
Honestly speaking I haven't looked into this in that detail; my focus was to repair the test and to add an assert to avoid such side effect again.
#26961 is still reproducible after #25094 was merged so there is for sure something left in this but this needs more thoughts.

Reply:
Just tried and it seems `HTTPCLIENTREQUEST` is never emitted. This is for sure a bug.